src/pages/Search/EmptySearchView.tsx

```diff
@@ -97,16 +94,10 @@ function EmptySearchView({type, hasResults}: EmptySearchViewProps) {
                         </View>
                     ))}
                 </View>
-                {!!ctaErrorMessage && (
-                    <DotIndicatorMessage
-                        style={styles.mt1}
-                        messages={{error: ctaErrorMessage}}
-                        type="error"
-                    />
-                )}
+                <BookTravelButton text={translate('search.searchResults.emptyTripResults.buttonText')} />
```

Comment:
I would also use `children` instead of a `text` prop as well. I think it's a shame that our normal `<Button>` component doesn't use `children`.

Reply:
Well, `BookTravelButton` wraps our normal `Button` so in order to add children, I need to refactor the `Button` as well, no?

Reply:
OK, just a NAB then.